src/tstl.ts

```diff
@@ -95,20 +96,27 @@ function performCompilation(
     options: tstl.CompilerOptions,
     configFileParsingDiagnostics?: readonly ts.Diagnostic[]
 ): void {
+    if (options.measurePerformance) performance.enable();
```

Comment:
I'd propose either renaming `performance` to `measurePerformance` (in all files), *or* to rename `enable` to `enableMeasurement` (and `disableMeasurement`). `performance.enable()` looks more like we're enabling some kind of performance mode, it's not clear.